receiver/k8sclusterreceiver/watcher.go

```diff
@@ -153,6 +176,16 @@ func (rw *resourceWatcher) prepareSharedInformerFactory() error {
 	}
 	rw.informerFactories = append(rw.informerFactories, factory)
 
+	isSupported, _ := rw.isKindSupported(gvk.HierarchicalResourceQuota)
```

Comment:
If you want to target other CRDs as well, it would be better to refactor it to pass gvr/gvk in a for loop.

Reply:
I fixed to handle errors.